tests/parker.js

```diff
@@ -12,7 +12,7 @@ function generateMetrics(file, metricsArray) {
     {
       name: 'Stylesheet size',
       benchmark: 150000,
-      threshold: 440000,
+      threshold: 445000,
```

Comment:
Maybe worth bumping it further, otherwise we will have to do it again soon enough.

Maybe 500k is enough of a threshold for us to be worried about (if at all)?

Reply:
Bumped to 500k